### PR TITLE
add skills warning

### PR DESCRIPTION
**Pull request by ernopp**
As discussed in [this doc](https://docs.google.com/document/d/1KzG2FAuAvGoXO8ej9d2uCaNPjba0XowDj2-luIhtpJk/edit#) 

Tested in improbadoc:

2018-12-19 17_04_14-improbadoc

I also driveby removed a dupe get-started.md that does not get linked to from anywhere from what I can see

### CODE REVIEW

File: docs/content/get-started/get-started.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Get started with SpatialOS for Unity by launching a cloud deployment of our FPS
 
 Once you’ve done this, you’ll then be ready to learn how to build your own features. Our tutorial for adding Health Pickups functionality will introduce you to this development experience.
 
+<%(Callout type="alert" message="
+Please note that basic proficiency of programming in Unity, version control systems, and command line interfaces is required to use the SpatialOS GDK for Unity.")%>
```
Comment (paulbalaji) [inline docs/content/get-started/get-started.md]: might be good to provide links to some Unity/git tutorials here

**Comment by paulbalaji** [comment]
could you also add some text in the setup that users can download a zipped bundle of the latest release if they are unfamiliar with git:
https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/latest

**Comment by ernopp** [comment]
Updated to include zip download option

cc @paulbalaji @firtina-improbable 

2018-12-21 15_49_28-improbadoc

File: docs/content/get-started/set-up.md [hunk 2]
```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
...
 
 
-## Clone the repos
+## Get the GDK and Starter Project Source Code
+
+### Zip Download
+
+If you are unfamiliar with Git, you can download a Zip file of the GDK and its Starter Project [here](https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3). 
+
+Once you have done that, you can skip the rest of this page and move on to [Open the FPS Starter Project]({{urlRoot}}/content/get-started/open-project.md).
```
Comment (paulbalaji) [inline docs/content/get-started/set-up.md]: Mention that they need to unzip the file before moving on

And maybe make this bit bold so people don't download a zip _and_ clone the repos (if they truly have no idea what they're doing)

File: docs/content/get-started/set-up.md [hunk 3]
```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
...
+
+<%(Callout type="alert" message="
+If you can, please use Git to get the GDK and its starter project, as version control will make it easier for you to get updates in the future.")%>
+
+### Cloning the Repos using Git
 
 You need to clone two repositories; the FPS Starter Project and the GDK for Unity.
```
Comment (paulbalaji) [inline docs/content/get-started/set-up.md]: maybe call out again here that if they have already downloaded the zip, they do not need to clone the repos as well

File: docs/content/get-started/set-up.md [hunk 4]
```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
...
 
-
 |     |     |
 | --- | --- |
-| SSH | `git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git` |
 | HTTPS | `git clone https://github.com/spatialos/gdk-for-unity-fps-starter-project.git` |
+| SSH | `git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git` |
+
+<%(Callout type="warning" message="
+We recommend using SSH - if you're not using SSH to connect to Github, follow [this handy guide](https://help.github.com/articles/connecting-to-github-with-ssh/) to set it up.")%>
```
Comment (paulbalaji) [inline docs/content/get-started/set-up.md]: I don't think we recommend something either way - I'd rephrase this to be more like

```
You can only clone via SSH if you have already [set up SSH keys](https://help.github.com/articles/connecting-to-github-with-ssh/) with your GitHub account.
```

File: docs/content/get-started/set-up.md (hunk 2 above)
Comment (paulbalaji) [reply docs/content/get-started/set-up.md]: perhaps put the second paragraph into an information callout
